src/components/common/Tabs/TabItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from 'styled-components'
+
+interface TabProps {
+  title: string
+  readonly id: number
+  onTabClick: (arg: number) => void
+  isActive: boolean
+  readonly activeColor?: string
+}
+
+interface WrapperProps {
+  readonly activeColor?: string
+  isActive: boolean
+}
+
+const Wrapper = styled.li<WrapperProps>`
+  background: ${(props): string => `var(${props.isActive ? props.activeColor : '--color-primary'})`};
```

Comment:
@W3stside Is this approach fine too? Wasn't sure if the `.attr` is required, unless I'm changing actual attributes of the element? E.g. data-%, type, className, etc.

But my hunch from your example, is that your store these variables to reuse in the CSS below. Let me know as I'm a bit blank on that specific syntax.

Reply:
it's not necessary no, this is fine :+1: